packages/mssql/mssql.1.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+authors: ["Arena Developers <silver-snakes@arena.io>"]
+maintainer: "silver-snakes@arena.io"
+homepage: "https://github.com/arenadotio/ocaml-mssql"
+dev-repo: "git+https://github.com/arenadotio/ocaml-mssql.git"
+bug-reports: "https://github.com/arenadotio/ocaml-mssql/issues"
+doc: "https://arenadotio.github.io/ocaml-mssql/doc"
+synopsis: "Async SQL Server client using FreeTDS"
+description: "Mssql wraps FreeTDS in a nicer and safer interface, with support for parameterized queries, thread-based async IO, and a thread pool."
+
+build: [
+  ["jbuilder" "build" "-p" name "-j" jobs]
```

Comment:
you seem to be using `dune` instead of jbuilder according to your source code. Is that correct? If so I think it would be slightly better to use the `dune`commands instead of `jbuilder`

Reply:
Running `dune` turns more warnings into errors so it won't actually build that way. I'll look into making that change in a future release but it's a non-trivial change.

Reply:
Actually maybe it's only a few warnings to fix..